src/lib/temple/activity-new/utils.ts

```diff
@@ -63,3 +111,21 @@ export const getActor = (activity: DisplayableActivity) => {
     actor: isSend || isDelegation ? activity.to : activity.from
   };
 };
+
+export const isQuipuswapSubtype = (subtype: ActivitySubtype): boolean =>
+  subtype === ActivitySubtype.QuipuswapCoinflipBet ||
```

Comment:
maybe we can create something like QuipuswapTypes[] and use Array.includes() method ?

Reply:
@lendihop but why not?